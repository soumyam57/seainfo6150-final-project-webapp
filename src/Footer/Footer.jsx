import React from 'react';
import styles from "./Footer.module.css";

const Footer = () => {
    return (
        <div className={styles.footer}>
            <div className={styles.margin}>
                <p>Soumya Murugendrappa</p>
                <p>A Web Design Project, @Northeastern University, Silicon Valley, CA, USA</p>
            </div>
        </div>
    )
}

export default Footer;
