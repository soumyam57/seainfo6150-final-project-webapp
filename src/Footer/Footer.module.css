.footer{
    padding-top: 3%;
    text-align: center;
    color:rgb(117, 104, 104);
    background-image: linear-gradient(to right,rgb(83, 15, 15), rgb(49, 12, 25) 70%);
    font-family: 'Space Grotesk', sans-serif;
    font-size: smaller;
}

.margin{
    border-top: 0.5px solid rgb(102, 18, 18);
    padding: 0.5%;
}