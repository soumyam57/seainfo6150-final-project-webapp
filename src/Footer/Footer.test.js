import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer.jsx";

it("renders the Footer component correctly", () => {
  const { container } = render(
    <Footer/>
    );
  expect(container).toMatchSnapshot();
});