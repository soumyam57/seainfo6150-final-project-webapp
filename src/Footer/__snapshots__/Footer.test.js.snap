// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Footer component correctly 1`] = `
<div>
  <div
    class="footer"
  >
    <div
      class="margin"
    >
      <p>
        Soumya Murugendrappa
      </p>
      <p>
        A Web Design Project, @Northeastern University, Silicon Valley, CA, USA
      </p>
    </div>
  </div>
</div>
`;
